warehouse/schema.go

```diff
@@ -13,12 +13,13 @@ import (
 )
 
 type SchemaHandleT struct {
-	dbHandle          *sql.DB
-	stagingFiles      []*StagingFileT
-	warehouse         warehouseutils.Warehouse
-	localSchema       warehouseutils.SchemaT
-	schemaInWarehouse warehouseutils.SchemaT
-	uploadSchema      warehouseutils.SchemaT
+	dbHandle                      *sql.DB
+	stagingFiles                  []*StagingFileT
+	warehouse                     warehouseutils.Warehouse
+	localSchema                   warehouseutils.SchemaT
+	schemaInWarehouse             warehouseutils.SchemaT
+	unRecognizedSchemaInWarehouse warehouseutils.SchemaT
```

Comment:
Should naming be `unrecognizedSchemaInWarehouse` ?

Reply:
Done.